redi-connect-front/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
We should avoid adding `.env` files to the repo as it's devbox-specific configuration and generally an anti-pattern. Consider creating a `.env.template` file that a developer can copy-paste from for local setup.

Also, let's keep some light documentation in `README.md` about environment variables that aren't self-documenting. I don't immediately see what this variable does.